bin/tictactoe

```diff
@@ -1,3 +1,30 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
for lines 4-30 I would convert this into a CLI class with callable instance methods that lives in the `lib` folder. Then on line 5 you could just call this: 
```ruby 
CLI.new.start
```